### PR TITLE
Update planning.rst

### PR DESCRIPTION
**Pull request by rhoughton**
1st change: Fixed small grammatical mistake. 
2nd change: Disambiguated the use of the word "it" into a more readable and digestable format. 
I love these docs, thank you for allowing me to contribute.

### CODE REVIEW

File: pages/getting_started/planning.rst [hunk 1]
```diff
@@ -75,7 +75,11 @@ Collection method
 ^^^^^^^^^^^^^^^^^
 **"How will you collect it?"**
  
-After a list of event sources has been determined, the next step is to decide the method of collection for each source.  Although many hardware and software products support common methods such as sending log data via syslog, many do not. It is critical to understand what method each event source uses and what resources that may require. For example, if a log shipper will be required to read logs from a local file on all servers, a log shipper must be selected and tested prior to deployment. In other cases, proprietary API’s or software tools must be employed and integrated.
+After a list of event sources has been determined, the next step is to decide the method of collection for each source.  Although many hardware and software products support common methods such as sending log data via syslog, many do not. 
+Understanding the answer to these questions is critical: 
+*What method does each event source use?
+*What resources are required?
```
Comment (asachs01) [inline pages/getting_started/planning.rst]:
```suggestion
* What method does each event source use?
* What resources are required? 
```

Comment (asachs01) [reply pages/getting_started/planning.rst]: @rhoughton thanks for contributing! Just a note that for bulleted lists to work, they'll need a space between the * and the rest of the characters.

**Review by asachs01 — CHANGES_REQUESTED** [review]

File: pages/getting_started/planning.rst (hunk 1 above)
Comment (rhoughton) [reply pages/getting_started/planning.rst]: oic, thank you! .rst files are cool, I'm learning :)

**Review by asachs01 — APPROVED** [review]
I've committed the changes @rhoughton, so we'll pull this in.

**Pull request merged**